src/public/app/features/team/team-create/team-create.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs/Observable';
+import { map, tap } from 'rxjs/operators';
+import { Team } from './team-create.component';
+import { LoggerService } from '../../../core/logger.service';
+
+@Injectable()
+export class TeamCreateService {
```

Comment:
Just a thought, but why not make this a TeamService and then it can be used by both the teams page and the create a team page?

Reply:
That seems logical to me.